internal/generator/services.go

```diff
@@ -244,6 +244,11 @@ func composeToServiceValues(
 			}
 		}
 
+		baseimage := lagoon.CheckServiceLagoonLabel(composeServiceValues.Labels, "lagoon.base.image")
+		if baseimage != "" {
+			buildValues.ForcePullImages = append(buildValues.ForcePullImages, baseimage)
```

Comment:
Just a thought, we're just adding the image here and then the success or failure of this happens during the build step.

Kubernetes does this using validations/reference, we could do that here and drop an error before the build even progresses if the image is bad: https://go.dev/play/p/YNAfkmMLFTh

Reply:
That would catch out a bad image name/tag definition earlier and throw an error then, rather than at the `docker pull` step

Reply:
done :)